src/Entity/CategoryManager.php

```diff
@@ -48,12 +47,6 @@ public function __construct($class, ManagerRegistry $registry, ContextManagerInt
 
     /**
      * Returns a pager to iterate over the root category.
-     *
-     * @param int   $page
```

Comment:
Why removing this? There is no typehint so this is the only place to get the type, isn’t it?

Reply:
The type hint was moved: https://github.com/sonata-project/SonataClassificationBundle/pull/463/commits/e5739d249bf334ddb3f49c6df79c804d12f2dd10#diff-4d9dd8b669cb2a016d76d86e7d6f0384R20